jobs_detector/helpers.py

```diff
@@ -0,0 +1,63 @@
+import re
+from collections import defaultdict
+
+def _keyword_search(comment, keyword):
+    """
+    Return True if the keyword is found in the search and add to keywords_dict
+    and comments_dict dictonaries.
+    
+    keywords_dict = {Python:[12345,09876,45677], 
+                     Remote:[12345,98752,15973]}
+                    
+    comments_dict = {12345:'This is the comment test of the post'}
+                    {98765:'This is another comment test of the post'}                
+    """
+    if re.search(re.compile(keyword, re.IGNORECASE), str(comment)):
```

Comment:
I don't think re is necessary just for matching a word in a comment, you could just say `if keyword in comment`. Also can you use BeautifulSoup instead of re to find the id below? That seems like it would be preferable than using re to parse web sites. 

I thought it was interesting you are getting the id, we didn't do that. But it would be useful if you wanted to log all the comments in a database or something.